### DISCUSSION TITLE
Test Interrupteur Sonoff ZBMINI-L2 extrême - Haade.fr

### DISCUSSION THREAD

**Discussion by giscus** [haade-administrator/haade.fr#25] [Comments]
# Test Interrupteur Sonoff ZBMINI-L2 extrême - Haade.fr

Test du nouvel interrupteur zigbee de chez Sonoff le ZBMINI-L2 extrême sans neutre, est-il si petit ? Que vaut-il ?

https://haade.fr/fr/blog/test-sonoff-zbminil2-extreme-zigbee-end-device

**Comment by pir2d2** [comment]
Bonjour et merci pour tes articles.
Sais tu comment câbler un va et vient existant. 
La phase arrive d'un côté, deux fils vont du premier au second interrupteur, puis le tout va vers la lumière puis le neutre du tableau elec.
Mais alors comment câbler le module pour s'insérer dans ce schéma sans ajouter un câble ?
Rien ne remplacera le fibaro dimmer zwave fgd-212

Tu as une idée ?
Pierre
